app/controllers/user_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UserController < ApplicationController
-  def new; end
+  def show
+    @user = User.find params[:id]
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render "new"
```

Comment:
```suggestion
      render :new
```
thêm 1 flash báo lỗi nữa nhé e